src/react-apps/applications/receipt/.env

```diff
@@ -0,0 +1 @@
+PORT=9000
```

Comment:
This port doesn't match what is defined in docker-compose.yaml or the Dockerfile. Is this file even used?

Reply:
Not used, deleting